src/regtech_data_validator/create_schemas.py

```diff
@@ -9,6 +9,13 @@
 from regtech_data_validator.phase_validations import get_phase_1_and_2_validations_for_lei
 from regtech_data_validator.schema_template import get_template
 
+from enum import Enum
+
+
+class ValidationPhase(str, Enum):
+    SYNTACTICAL = "Syntactical"
+    LOGICAL = "Logical"
```

Comment:
Elsewhere in the code we use [`enum.StrEnum`](https://docs.python.org/3/library/enum.html#enum.StrEnum). This _may_ end up being equivalent, but probably want to switch just for consistency.

https://github.com/cfpb/regtech-data-validator/blob/63b4735b85aca3837e7d7da0f587ac2580e0ab49/src/regtech_data_validator/checks.py#L13-L15

Reply:
Ah in all the other repos we do (str, Enum), StrEnum is a 3.11 and beyond thing so maybe that's why.  I'll swap here, might be good to move them all over to StrEnum at some point